service/controller/v14/adapter/guest_lifecycle_hooks.go

```diff
@@ -0,0 +1,32 @@
+package adapter
+
+import "github.com/giantswarm/aws-operator/service/controller/v14/key"
+
+// The template related to this adapter can be found in the following import.
+//
+//     github.com/giantswarm/aws-operator/service/controller/v14/templates/cloudformation/guest/guestLifecycle_hooks.go
+//
```

Comment:
These comments got removed in other places. You might want to double check so we treat this the same way everywhere.

Reply:
I just did it by best effort to release the PR quicker. I'll iterate again. I don't think those comments are that useful now when we have all scoped. Also they are pain to maintain.

Reply:
Done